src/components/Footer.astro

```diff
@@ -16,9 +16,10 @@ import Icon from "./Icon.astro"
 
 const navigation = {
   support: [
-    { name: "Documentation", href: DOCUMENTATION_URL },
-    { name: "GitHub Issues", href: GITHUB_ISSUES_URL },
+    { name: "Docs", href: DOCUMENTATION_URL },
+    { name: "Issues", href: GITHUB_ISSUES_URL },
```

Comment:
```suggestion
    { name: "Docs", href: DOCUMENTATION_URL },
    { name: "Issues", href: GITHUB_ISSUES_URL },
```

Is there a reason to shorten “Documentation”, and make “GitHub Issues” _less_ specific when that’s still what it links to?

Reply:
The longer word "Documentation" messed up the footer columns when there are 4 of them. I imagine you know something better to do than shorten the word, happy to know what to do!

Reply:
I intentionally used the longer word in the footer since the shortened version was already in the header. I’d always have the page layout support the content and not the other way around, so adjust the columns instead. (If there needs to be another column, per other comments.)

Reply:
Tried changing it back to "Documentation" but don't like it, happy to have a suggestion that will make it work. 

<img width="414" alt="image" src="https://github.com/ddev/ddev.com-front-end/assets/112444/c8345313-98f1-439e-8903-6d45629f6d5b">